### PR TITLE
Add Redque connector

### PR DESCRIPTION
**Pull request by jirisodomka**
- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

**Comment by jirisodomka** [comment]
@microsoft-github-policy-service agree [company="{Redque s.r.o.}"]

**Comment by jirisodomka** [comment]
@microsoft-github-policy-service agree company="Redque s.r.o."

File: certified-connectors/Redque/Intro.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Prerequisites
```
Comment (Amjed-Ayoub) [inline certified-connectors/Redque/Intro.md:1]: @jirisodomka thank you so much for your submission. May I ask you please to remove "Intro.md" and "redque.png" as they are not required in the open source repository.

File: certified-connectors/Redque/Readme.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* `Remove token`: Removes registration token
+### Users
+* `Filter users`: Gets the list of user by criteria
+* `Change password`: Changes user's password
+* `Change user's password`: Changes password for specified user. Requires user management permission
+* `Delete user`: Deletes user
+* `Find user`: Finds user
+* `Update user`: Partially updates specified user
+* `Add permissions`: Grants permissions
+* `Remove permissions`: Revokes document permissions
```
Comment (Amjed-Ayoub) [inline certified-connectors/Redque/Readme.md:58]: @jirisodomka please add all required sections from this [template](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors).
Your readme file is missing the below sections:
```
## Obtaining Credentials
Required. Explain the authentication method and how to get the credentials.​

## Known Issues and Limitations
Required. Include any known issues and limitations a user may encounter.

## Deployment Instructions
Required. Add instructions on how to deploy this connector as custom connector.
```

File: certified-connectors/Redque/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,3548 @@
+{
+    "swagger": "2.0",
+    "basePath": "/",
+    "consumes": [],
+    "host": "demoidentity.redque.com",
+    "info": {
+        "title": "Redque",
+        "description": "Accessing and managing the Redque system",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Redque/apiDefinition.swagger.json:8]: @jirisodomka please add a period at the end of the description. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

File: certified-connectors/Redque/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,3548 @@
...
+                "value": {
+                    "type": "string"
+                }
+            }
+        }
+    },
+    "x-ms-connector-metadata": [
+        {
+            "propertyName": "Website",
+            "propertyValue": "https://redque.com"
```
Comment (Amjed-Ayoub) [inline certified-connectors/Redque/apiDefinition.swagger.json:3518]: @jirisodomka the website URL should be a valid URL. please change to "https://redque.cz/en/" or "https://redque.cz"

File: certified-connectors/Redque/apiProperties.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+          "displayName": "Log in with RedQue Account",
+          "description": "Log in with RedQue Account",
+          "tooltip": "Log in with RedQue Account",
+          "constraints": {
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0db9d6",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Redque/apiProperties.json:43]: @jirisodomka this background color does not match with your icon. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector)

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/Redque/Readme.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+* `Get image`: Gets link to an image of a given page
+* `Return to issuer`: Returns document to the issuer for corrections
+* `List documents`: List filtered documents based on parameters
+* `Export document`: Export document to accounting provider
+* `Donwload document`: Downloads document's file or gets URL
+* `Get document's metadata`: Gets document's metadata
+* `Update document`: Updates document
+* `Lock document`: Locks (opens) document preventing other users from modifying it
+* `Unlock document`: Unlocks locked (closes) document to allow other users to modify its contents
+* `Grant access`: Grants document access to specified user
```
Comment (Amjed-Ayoub) [inline certified-connectors/Redque/Readme.md:27]: @jirisodomka all files should be free of grammatical errors and typos. please change "acces" to "access". You have the same typos in your readme and swagger file as well please fix both files.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/Redque/Intro.md (hunk 1 above)
Comment (jirisodomka) [reply certified-connectors/Redque/Intro.md:1]: Hello @Amjed-Ayoub thank you for the review. I will do necessary changes you mentioned.

File: certified-connectors/Redque/apiDefinition.swagger.json (hunk 3 above)
Comment (jirisodomka) [reply certified-connectors/Redque/apiDefinition.swagger.json:8]: @Amjed-Ayoub You mean in the sentence? But in the description, you mentioned is sentence without period, right?
image
Or did you mean different description? Anyway, I will update it here.  Thank you.

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @jirisodomka,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**